tasks/greenlight.yml

```diff
@@ -35,8 +35,18 @@
   - greenlight-redirect.nginx
   notify: reload nginx
 
-- name: start greenlight
-  docker_compose:
-    pull: true
-    project_src: /root/greenlight/
+- block:
+    - name: start greenlight
+      docker_compose:
+        pull: true
+        project_src: /root/greenlight/
+
+    # https://docs.bigbluebutton.org/greenlight/gl-admin.html#creating-accounts
+    - name: Create greenlight users
+      command: "docker exec greenlight-v2 bundle exec rake user:create[\"{{ item.name }}\",\"{{ item.email }}\",\"{{ item.password }}\",\"{{ item.type }}\"]"
+      with_items: "{{ bbb_greenlight_users }}"
```

Comment:
Please use `loop` here
https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#migrating-from-with-x-to-loop